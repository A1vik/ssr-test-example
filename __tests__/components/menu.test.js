import React from "react";
import Menu from "../../src/shared/components/menu";
import { shallow } from "enzyme";

describe("Menu", () => {
  test("match snapshot", () => {
    const wrapper = shallow(<Menu/>);
    expect(wrapper).toMatchSnapshot();
  });
});
