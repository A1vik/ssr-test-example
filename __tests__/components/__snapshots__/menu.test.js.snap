// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu match snapshot 1`] = `
<ul>
  <li
    key="us"
  >
    <NavLink
      activeClassName="active"
      activeStyle={
        Object {
          "fontWeight": "bold",
        }
      }
      aria-current="page"
      to="/popular/us"
    >
      US
    </NavLink>
  </li>
  <li
    key="ae"
  >
    <NavLink
      activeClassName="active"
      activeStyle={
        Object {
          "fontWeight": "bold",
        }
      }
      aria-current="page"
      to="/popular/ae"
    >
      AE
    </NavLink>
  </li>
  <li
    key="cz"
  >
    <NavLink
      activeClassName="active"
      activeStyle={
        Object {
          "fontWeight": "bold",
        }
      }
      aria-current="page"
      to="/popular/cz"
    >
      CZ
    </NavLink>
  </li>
  <li
    key="de"
  >
    <NavLink
      activeClassName="active"
      activeStyle={
        Object {
          "fontWeight": "bold",
        }
      }
      aria-current="page"
      to="/popular/de"
    >
      DE
    </NavLink>
  </li>
  <li
    key="be"
  >
    <NavLink
      activeClassName="active"
      activeStyle={
        Object {
          "fontWeight": "bold",
        }
      }
      aria-current="page"
      to="/popular/be"
    >
      BE
    </NavLink>
  </li>
</ul>
`;
